big-practice/src/assets/css/update-customer.css

```diff
@@ -0,0 +1,174 @@
+.modal-update {
+  width: 968px;
+  height: 797px;
+  position: absolute;
+  top: 290%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1px solid black;
```

Comment:
Should use hexadecimal color code for color variables.
`border: 1px solid black;` => `border: 1px solid #000`;

Reply:
My mistake, I'm sorry. I just try to check and I forgot to change it.